services/bit/bit-components.service.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+const Joi = require('joi')
+const { BaseJsonService } = require('..')
+const { metric } = require('../text-formatters')
+
+const ownerSchema = Joi.object({
+  payload: Joi.object({
+    totalComponents: Joi.number(),
+  }),
+}).required()
+
+module.exports = class bitComponents extends BaseJsonService {
+  async fetch({ owner, scope }) {
+    const url = `https://api.bit.dev/scope/${owner}/${scope}/`
```

Comment:
Is there any public documentation available for the API? If so it would be great to include the link in an inline comment (`// API Docs - https://docs.bit.dev/... `) to make it easier for folks to find that info.

Reply:
there is no official API documentation.

Reply:
> there is no official API documentation

Ah that's disappointing but thanks for confirming. Do you know if any docs are pending? Or is this an unofficial/unsupported API? 

I ask because Shields is maintained by a small number of volunteers, but it supports hundreds of services. As such we like to have as much info available as possible to make it easier for us to maintain/support all those various integrations.

Reply:
I think the official docs for the API will publish until the end of this year,
but I will need to confirm it, and update you.

Reply:
Thanks! No need to block on that, but feel free to drop us a note (or a PR 😄) when those docs become public